fundingmanager_test.go

```diff
@@ -1131,7 +1163,8 @@ func TestFundingManagerOfflinePeer(t *testing.T) {
 	close(con)
 
 	// This should make Alice send the fundingLocked.
-	fundingLockedAlice := checkNodeSendingFundingLocked(t, alice)
+	fundingLockedAlice :=
+		assertFundingMsgSent(t, alice.msgChan, "FundingLocked").(*lnwire.FundingLocked)
```

Comment:
Really like how this turned out! I might suggest wrapping the lines like this:
```golang
fundingLockedAlice := assertFundingMsgSent(
        t, alice.msgChan, "FundingLocked",
).(*lnwire.FundingLocked)
```

feel free to do this on similar lines that are over 80 chars

Reply:
Done. Pretty sure I got em all.